branching.md

```
@@ -1 +1,5 @@
 #This is the branching strategy that you need to follow.
```

Markdownlint

No space after hash on atx style heading

(MD018, no-missing-space-atx)